doc/computation.rst

```diff
@@ -225,6 +225,23 @@ If the result would be empty, an error is raised instead:
     In [1]: arr[:2] + arr[2:]
     ValueError: no overlapping labels for some dimensions: ['x']
 
+However, one can explicitly change this default automatic alignment type ("inner")
+via :py:func:`~xarray.set_options()`
+
+.. ipython:: python
+
+    xr.set_options(arithmetic_join="outer")
```

Comment:
For our use case we would set `arithmetic_join="outer"` from the get-go and run all of our binary calculations like that throughout our simulation.  We'd use that as our default, and maybe occasionally switch to other join types.  Requiring context manager every time we deviate from "inner" seems onerous.